cmd/lotus-miner/sectors.go

```diff
@@ -770,9 +769,20 @@ func getSectorsFromFile(filePath string) ([]uint64, error) {
 	return sectors, nil
 }
 
+func SectorNumsToBitfield(sectors []abi.SectorNumber) bitfield.BitField {
+	var numbers []uint64
+	for sector := range sectors {
+		numbers = append(numbers, uint64(sector))
+	}
+
+	return bitfield.NewFromSet(numbers)
+}
+
 var sectorsRenewCmd = &cli.Command{
-	Name:  "renew",
-	Usage: "Renew expiring sectors while not exceeding each sector's max life",
+	Name:      "renew",
+	Usage:     "Renew expiring sectors while not exceeding each sector's max life",
+	ArgsUsage: "<sectorNumbers...(optional)>",
+	Aliases:   []string{"extend"},
```

Comment:
I believe it's fine to not have the alias, possibly even preferred. Flags won't match, so it is a breaking change either way